cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -314,8 +314,8 @@ func TestPolicyUpdate(t *testing.T) {
 					adsysSymlink: currentUser,
 				},
 				{
-					src:          "UserIntegrationTest@example.com.krb5",
-					adsysSymlink: "UserIntegrationTest@example.com",
+					src:          "userintegrationtest@example.com.krb5",
+					adsysSymlink: "userintegrationtest@example.com",
```

Comment:
Same here, we can have a dedicated non lowercase tests (we should have that on any non integration tests with user input)

Reply:
Ah, so it’s either the mock to adapt or see what’s happening with KRB5CC file? Can you try to check we DTRT regarding KRB5CCNAME when creating the symlink? I’m pretty sure this all depends on ourself.